source/helix-ui.less

```diff
@@ -46,3 +47,8 @@
 .text-nowrap {
   white-space: nowrap;
 }
+
+/* ===== Helix Helpers ===== */
+.hxWordBreak {
```

Comment:
I'm inclined to name this `.hxForceNewline` to clearly describe its intended use. As it stands, it's just mirroring the property name itself.

Reply:
@parlarjb: any thoughts?

Reply:
FYI: moving forward with `.hxForceWordBreak`

Reply:
I like `.hxForceWordBreak` better than `.hxForceNewline`